src/modules/cmdpal/.wt.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
Oh this one's actually a new-ish Terminal feature - you can stick commandline snippets in a directory to have them displayed in the Terminal. This one we should keep ☺️

Reply:
oh, ok.. didn't know that :) Reverting